### ISSUE TITLE
Many npm packages are old and npm update breaks

### ISSUE THREAD

**Issue by peterschriever** [hyva-themes/magento2-hyva-checkout#102]
Possibly related to #101 

Many of the packages in package.json are outdated, not just in terms of major version release but also minor versions. The command `npm update` which is to update all the packages dictated by their configured tag breaks most likely to a similar issue as npm install without package-lock (#101)

```bash
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: hyva-checkout@0.1.0
npm ERR! Found: eslint@7.32.0
npm ERR! node_modules/eslint
npm ERR!   dev eslint@"^7.0.0" from the root project
npm ERR!   peer eslint@"^5.16.0 || ^6.8.0 || ^7.2.0" from eslint-config-airbnb@18.2.1
npm ERR!   node_modules/eslint-config-airbnb
npm ERR!     dev eslint-config-airbnb@"^18.0.1" from the root project
npm ERR!   3 more (eslint-plugin-import, eslint-plugin-jsx-a11y, eslint-plugin-react)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer eslint@"^3.0.0 || ^4.0.0 || ^5.0.0 || ^6.0.0" from eslint-plugin-react-hooks@1.7.0
npm ERR! node_modules/eslint-plugin-react-hooks
npm ERR!   dev eslint-plugin-react-hooks@"^1.7.0" from the root project
npm ERR!   peer eslint-plugin-react-hooks@"^4 || ^3 || ^2.3.0 || ^1.7.0" from eslint-config-airbnb@18.2.1
npm ERR!   node_modules/eslint-config-airbnb
npm ERR!     dev eslint-config-airbnb@"^18.0.1" from the root project
```

Bug fixes and vulnerability patches can not be applied unless done manually at this moment!

**Comment by progammer-rkt** [comment]
Thank you for reporting this. This is a valid concern. I updated them to the latest and the work is available in #130

**Comment by progammer-rkt** [comment]
The fix reached `develop` branch and hence closing this issue
